can-compute-stream.js

```diff
@@ -1,80 +1,163 @@
 var compute = require('can-compute');
-var define = require('can-define');
 var Kefir = require('kefir');
 var makeArray = require("can-util/js/make-array/make-array");
 var isArray = require("can-util/js/is-array/is-array");
 var assign = require("can-util/js/assign/assign");
+var define = require('can-define');
 
+// Pipes the value of a compute into a stream
 var singleComputeToStream = function (compute) {
 	return Kefir.stream(function (emitter) {
 		var changeHandler = function (ev, newVal) {
 			emitter.emit(newVal);
 		};
 
-		emitter.emit(compute());
 		compute.on('change', changeHandler);
+		emitter.emit(compute());
 
 		return function () {
 			compute.off('change', changeHandler);
 		};
 	});
 };
 
+// Converts all arguments passed to streams, and resolves the resulting
+// streams to a single stream
+//   Assumes all arguments are computes
+//   Last argument is optionally a function
 var computeStream = function () {
```

Comment:
I think this should be renamed something else.  Why is this function the export?  I'd expect an object to be exported.